frontend/src/metabase/components/Icon.jsx

```diff
@@ -62,6 +62,10 @@ class BaseIcon extends Component {
       return <span />;
     }
 
+    const { ariaLabel } = icon.attrs;
+    // must be removed to avoid "Unknown prop" warning
+    delete icon.attrs.ariaLabel;
```

Comment:
I might rewrite this as `const { ariaLabel, ...otherAttrs } = icon.attrs;` and then use `otherAttrs` rather than deleting `ariaLabel` from `icon.attrs`.

I think the mutating `delete` _is_ safe here, but it still makes me a little worried since the definition of `attrs` isn't visible. If that were defined once at the top level, deleting `ariaLabel` might break future invocations.